src/postgres-util/src/lib.rs

```diff
@@ -66,7 +77,9 @@ pub async fn publication_info(
     conn: &str,
     publication: &str,
 ) -> Result<Vec<TableInfo>, anyhow::Error> {
-    let (client, connection) = tokio_postgres::connect(&conn, NoTls).await?;
+    let config = conn.parse::<Config>()?;
```

Comment:
Is this type annotation necessary?

Reply:
That was a left-over. Thanks for catching 👍🏼